docs/modules/operation/pages/service-assurance/monitors/Pop3Monitor.adoc

```diff
@@ -1,14 +1,14 @@
 
 = Pop3Monitor
 
-The _POP3_ monitor tests for _POP3_ service availability on a node.
-The monitor first tries to establish a _TCP_ connection on the specified port.
-If a connection is established, a service banner should have been received.
-The monitor makes sure the service banner is a valid _POP3_ banner (ie: starts with `+OK`).
-If the banner is valid, the monitor sends a `QUIT` _POP3_ command and makes sure the service answers with a valid response (ie: a response that starts with `+OK`).
-The service is considered available if the service's answer to the `QUIT` command is valid.
+The POP3 monitor tests for POP3 service availability on a node.
+The monitor first tries to establish a TCP connection on the specified port.
+If it establishes a connection, a service banner should have been received.
+The monitor makes sure the service banner is a valid POP3 banner (in other words, starts with +OK).
+If the banner is valid, the monitor sends a QUIT POP3 command and makes sure the service answers with a valid response (a response that starts with +OK).
+The service is considered available if the service's answer to the QUIT command is valid.
 
-The behavior can be simulated with `telnet`:
+Simulate the behavior with telnet:
 
  $ telnet mail.opennms.org 110
```

Comment:
```suggestion
[source, bash]
----
 $ telnet mail.opennms.org 110
```

Reply:
We have removed a prompt to make it c&p friendly. The console doesn't have formatting. The bash script formatting is sometimes weird with Linux shell commands and arguments.

```
[source, console]
----
telnet mail.opennms.org 110
----
```